newton/_src/solvers/kamino/_src/linalg/conjugate.py

```
@@ -192,15 +192,15 @@ def check_termination(
     cur_iter: wp.array[int],
     world_condition: wp.array[wp.int32],
     batch_condition: wp.array[wp.int32],
 ):
     wid = wp.tid()
 
     # Update iteration
-    active = world_active[wid]
+    active = wp.int32(world_active[wid])
     condition = world_condition[wid]
     world_stepped = active * condition
     iter = world_stepped * loop_granularity + cur_iter[wid]
```

Ruff (0.15.14)

[error] Variable `iter` is shadowing a Python builtin

(A001)